scalalib/overrides-2.12/scala/collection/immutable/Range.scala

```diff
@@ -508,26 +519,29 @@ object Range {
     implicit val doubleAsIntegral = scala.math.Numeric.DoubleAsIfIntegral
     def toBD(x: Double): BigDecimal = scala.math.BigDecimal valueOf x
 
-    def apply(start: Double, end: Double, step: Double) =
+    def apply(start: Double, end: Double, step: Double): NumericRange[Double] =
```

Comment:
The mapRange forces the datatype to NumericRange vs NumericRange.Exclusive, not sure if that was intended